src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -1012,7 +1012,15 @@ public virtual object Clone()
                 ci._textInfo = (TextInfo)_textInfo.Clone();
             }
 
-            if (_calendar != null)
+            if (_dateTimeInfo != null && _dateTimeInfo.Calendar == _calendar)
+            {
+                // Usually when we access CultureInfo.DateTimeFormat first time, we create the DateTimeFormatInfo object
+                // using CultureInfo.Calendar. i.e. CultureInfo.DateTimeInfo.Calendar == CultureInfo.calendar.
+                // When cloning CultureInfo, if we know this still the case CultureInfo.DateTimeInfo.Calendar == CultureInfo.calendar
```

Comment:
Nit: "if we know this still the case" => "if we know it's still the case that"

Reply:
Sorry, I already merged the change. I can try to fix that next time I change something in coreclr. thanks for the comment.